src/main/java/org/jnode/fs/ntfs/datarun/CompressedDataRun.java

```diff
@@ -97,15 +97,15 @@ public int readClusters(long vcn, byte[] dst, int dstOffset, int nrClusters, int
         final long myLastVcn = getLastVcn();
         final long reqLastVcn = vcn + nrClusters - 1;
 
-        if (log.isDebugEnabled()) {
-            log.debug("me:" + myFirstVcn + "-" + myLastVcn + ", req:" + vcn + "-" + reqLastVcn);
-        }
-
         if ((vcn > myLastVcn) || (myFirstVcn > reqLastVcn)) {
             // Not my region
             return 0;
         }
 
+        if (log.isDebugEnabled()) {
+            log.debug("me:{}-{}, req:{}-{}", myFirstVcn, myLastVcn, vcn, reqLastVcn);
+        }
```

Comment:
This is moved below the range check to avoid the logs spewing out messages like:
```
DEBUG org.jnode.fs.ntfs.datarun.CompressedDataRun - me:0-15, req:3072-3087
DEBUG org.jnode.fs.ntfs.datarun.CompressedDataRun - me:16-31, req:3072-3087
DEBUG org.jnode.fs.ntfs.datarun.CompressedDataRun - me:32-47, req:3072-3087
... lots more up until cluster >= 3072
```